src/Workspaces/Core/Portable/Shared/TestHooks/IAsynchronousOperationListenerProvider.cs

```diff
@@ -127,7 +127,7 @@ public IAsynchronousOperationWaiter GetWaiter(string featureName)
         /// loop, dig into the waiters and see all of the active <see cref="IAsyncToken"/> values 
         /// representing the remaining work.
         /// </remarks>
-        public async Task WaitAllAsync(Workspace? workspace, string[]? featureNames = null, Action? eventProcessingAction = null, TimeSpan? timeout = null)
+        public async Task WaitAllAsync(Workspace? workspace, string[]? featureNames = null, TimeSpan? timeout = null)
```

Comment:
❗ This will break Apex unless we switch it to use the external access layer.

Reply:
We'll switch Apex after we do this.

Reply:
It should only be a binary breaking change; source should keep building since it doesn't pass anything a named parameter and doesn't pass positionally either. If it does break the source build, then we'll use that opportunity to move to the external access layer. :smile: